exercises/meetup/.meta/src/reference/java/Meetup.java

```diff
@@ -2,7 +2,7 @@
 import java.time.DayOfWeek;
 
 
-public class Meetup {
+class Meetup {
     private final LocalDate startOfMonth;
 
     public Meetup(int monthOfYear, int year) {
```

Comment:
This can also be package private :)

Reply:
Sorry for missing that one @FridaTveit, thought I got them all. Thanks for the feedback on this. I will get that one modified in a couple hours when I am in front of my computer again and modify the PR.

Reply:
Perfect!